CMakeLists.txt

```diff
@@ -143,6 +143,20 @@ else()
   find_package(GLEW REQUIRED)
 endif()
 
+# Check that Asio is installed. Asio does not have (or need) a Find… CMake Module, so we "only" need to check if the header exists.
+IF(NOT asio_location OR asio_location STREQUAL "")
+  unset(asio_location CACHE)
+  find_path(asio_location "asio.hpp")
+  if(asio_location)
+    message(STATUS "Found Asio in '${asio_location}'.")
+    get_filename_component(asio_include_dir ${asio_location} DIRECTORY)
+    list(APPEND CMAKE_INCLUDE_PATH ${asio_include_dir})
+  else()
+    message(FATAL_ERROR "Could NOT find Asio (missing: asio.hpp). Please install the standaline Asio library from https://think-async.com/Asio/")
```

Comment:
I think we should add the instructions here too, or to the wiki and link it here.